docs/pbc/all/payment-service-providers/payone/integrate-payone.md

```diff
@@ -40,5 +40,30 @@ To integrate Payone, follow these steps.
 If the app was connected successfully, a corresponding message appears, and the app status changes to **Connected**. The payment methods you've selected in step 8, appear in **Administration&nbsp;<span aria-label="and then">></span>  Payment methods**:
 ![payone-credit-card](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-credit-card.png).
 
+## 3. Whitelist Payone Domain
+
+To enable Payone to redirect your customers to their 3D Secure page and later one to your success page, you must add Payone domain inside your **Content Security Policy**. To do that please change `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

Comment:
```suggestion
To enable Payone to redirect your customers to their 3D Secure page and later one to your success page, you must add ACP domain inside your **Content Security Policy** whitelist. To do that please change `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

The page isn't owned by PayOne, but is owned by us. I suppose this does not matter too much for end users.